src/store/login/useAuthStore.js

```
@@ -1,6 +1,6 @@
 import { create } from 'zustand'
 import ROUTER_PATHS from '../../routes/RouterPath'
-import { postLoginData } from './../../domains/login/services/useLoginServices'
+import { deleteLogout, postLoginData } from './../../domains/login/services/useLoginServices'
```

ESLint

[error] 'deleteLogout' is defined but never used.

(no-unused-vars)